public/js/components/App.js

```diff
@@ -60,8 +60,8 @@ const App = React.createClass({
   },
 
   componentDidMount() {
-    this.shortcuts = new KeyShortcuts({ window });
-    this.shortcuts.on("CmdOrCtrl+P", this.toggleSourcesSearch);
+    const shortcuts = setupShortcuts(window);
```

Comment:
Thought a bit more about it and, it shouldn't be a big diff, so i'll take a look at contexts now.